src/lib.rs

```diff
@@ -560,6 +560,11 @@ impl Level {
     pub fn as_str(&self) -> &'static str {
         LOG_LEVEL_NAMES[*self as usize]
     }
+
+    /// Iterate through all supported logging levels
+    pub fn iter() -> impl Iterator<Item = Self> {
```

Comment:
It might be worth documenting something about the order of these fields. Using `from_usize` we’re effectively returning them in priority order.

What do you think?